cypress/mocks/sirius.json

```diff
@@ -0,0 +1 @@
ody":"{\"CT-bb\":{\"inserts\":[],\"location\":\"complaints/CT-bb.html.twig\",\"onScreenSummary\":\"CTBBONSCREENSUMMARY\"}}"}},{"name":"A request to create a payment","request":{"method":"POST","url":"/lpa-api/v1/cases/800/payments"},"response":{"status":201,"headers":{"Content-Type":"application/json"}}},{"name":"A request to edit the document","request":{"method":"PUT","url":"/lpa-api/v1/documents/dfef6714-b4fe-44c2-b26e-90dfe3663e95"},"response":{"status":200,"headers":{"Content-Type":"application/json"},"body":"{\"childCount\":0,\"content\":\"<p>Edited test content</p>\",\"createdDate\":\"15/12/2022 13:41:04\",\"direction\":\"Outgoing\",\"filename\":\"LP-A.pdf\",\"friendlyDescription\":\"Dr Consuela Aysien - __LPAONSCREENSUMMARY__\",\"id\":1,\"mimeType\":\"application\\\\/pdf\",\"systemType\":\"LP-A\",\"type\":\"Draft\",\"uuid\":\"dfef6714-b4fe-44c2-b26e-90dfe3663e95\"}"}},{"name":"A request to add a document to the case","request":{"method":"POST","url":"/lpa-api/v1/lpas/800/documents"},"response":{"status":201,"headers":{"Content-Type":"application/json"},"body":"{\"childCount\":0,\"content\":\"Test content\",\"createdDate\":\"15/12/2022 13:41:04\",\"direction\":\"Outgoing\",\"filename\":\"LP-A.pdf\",\"friendlyDescription\":\"Dr Consuela Aysien - __LPAONSCREENSUMMARY__\",\"id\":1,\"mimeType\":\"application\\\\/pdf\",\"systemType\":\"LP-A\",\"type\":\"Save\",\"uuid\":\"dfef6714-b4fe-44c2-b26e-90dfe3663e95\"}"}},{"name":"A request to delete a document","request":{"method":"DELETE","url":"/lpa-api/v1/documents/dfef6714-b4fe-44c2-b26e-90dfe3663e95"},"response":{"status":204,"headers":{}}},{"name":"A request to delete a payment","request":{"method":"DELETE","url":"/lpa-api/v1/payments/123"},"response":{"status":204,"headers":{}}},{"name":"A request to delete a fee reduction","request":{"method":"DELETE","url":"/lpa-api/v1/payments/124"},"response":{"status":204,"headers":{}}},{"name":"A request to create a task","request":{"method":"POST","url":"/lpa-api/v1/cases/800/tasks"},"response":{"status":201,"headers":{"Content-Type":"application/json"}}},{"name":"A request to create a task for a team","request":{"method":"POST","url":"/lpa-api/v1/cases/800/tasks"},"response":{"status":201,"headers":{"Content-Type":"application/json"}}},{"name":"A request to create a person reference","request":{"method":"POST","url":"/lpa-api/v1/persons/189/references"},"response":{"status":201,"headers":{"Content-Type":"application/json"}}},{"name":"A request to create a warning","request":{"method":"POST","url":"/lpa-api/v1/warnings"},"response":{"status":201,"headers":{"Content-Type":"application/json"}}},{"name":"A search for the deleted case","request":{"method":"GET","url":"/lpa-api/v1/deleted-cases?uid=700000005555"},"response":{"status":200,"headers":{"Content-Type":"application/json"},"body":"[{\"deletedAt\":\"02/12/2022\",\"deletionReason\":\"LPA was not paid for after 12 months\",\"status\":\"Return - unpaid\",\"type\":\"LPA\",\"uId\":\"7000-0000-5555\"}]"}}]}
```

Comment:
Is there any way to carry state between responses to mimic data stored in the session? This was something I needed to do for the govuk payments mock I wrote for Make. Also, is it possible to inject variables coming from the request back into the response? (Again, something I needed for govuk payment mocks.)